### PR TITLE
1 captcha

### PR DESCRIPTION
**Pull request by MohammadMokhtarii**
Add Captcha Feature With Multiple Cache Provider

### CODE REVIEW

File: src/Spyro.Core/Cache/InMemoryCache.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                cacheItemPolicy.AbsoluteExpiration = ExpireTime;
+            if (expirationType == ExpirationType.Sliding)
+                cacheItemPolicy.SlidingExpiration = ExpireTime.TimeOfDay;
+
+            cache.Set(cacheItem, cacheItemPolicy);
+        }
+    }
+
+
+    public enum ExpirationType
```
Comment (MohammadMokhtarii) [inline src/Spyro.Core/Cache/InMemoryCache.cs:53]: export to new file

File: src/Spyro.Core/Spyro.Core.csproj [hunk 2]
```diff
@@ -15,4 +15,9 @@
 		<PackageLicenseExpression>LGPL-3.0-or-later</PackageLicenseExpression>
 	</PropertyGroup>
 
+	<ItemGroup>
+	  <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="6.0.1" />
```
Comment (MohammadMokhtarii) [inline src/Spyro.Core/Spyro.Core.csproj:19]: must delete this dependency

**Review by behdadkd — APPROVED** [review]

**Review by behdadkd — APPROVED** [review]

**Review by behdadkd — APPROVED** [review]
its ok

**Review by behdadkd — CHANGES_REQUESTED** [review]
its ok

**Review by behdadkd — APPROVED** [review]
ok

**Pull request merged**